clearwater-infrastructure/usr/bin/health-tool

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
```

Comment:
Could you move health_tool_source into another directory (/usr/share/clearwater/health_tool?) and have this script just call /usr/share/clearwater/health_tool/run.py?